googlemock/test/gmock-more-actions_test.cc

```diff
@@ -325,7 +325,7 @@ TEST(InvokeTest, FunctionThatTakes10Arguments) {
 
 // Tests using Invoke() with functions with parameters declared as Unused.
 TEST(InvokeTest, FunctionWithUnusedParameters) {
-  Action<int(int, int, double, const string&)> a1 =
+  Action<int(int, int, double, const string)> a1 =
```

Comment:
The 'const' in a function argument type is meaningless.

Can you change the .travisci.yml and travis.sh scripts to prove out this fix?
It would good to have automatic checking of the osx c++11 build that you say is broken.